package.json

```diff
@@ -12,7 +12,7 @@
     "postbuild": "corepack yarn describe-version",
     "describe-version": "(git describe > version.txt.tmp && mv version.txt.tmp version.txt) || true && rm -f version.txt.tmp",
     "remove-tests-from-lib": "rm -rf lib/test/ && cp -r lib/src/* lib/ && rm -rf lib/src/",
-    "lint": "corepack yarn eslint src test && corepack yarn prettier --check src test",
+    "lint": "corepack yarn eslint --cache --quiet src test && corepack yarn prettier --cache --ignore-unknown --check src test",
```

Comment:
Why do we want eslint to be quiet?

Reply:
It was quiet until introduction of prettier, so I assumed unquieting as a mistake, but have now unquieted it again.

Reply:
Thankies